src/frequenz/sdk/actor/_power_managing/_bounds_methods.py

```diff
@@ -0,0 +1,143 @@
+# License: MIT
+# Copyright © 2023 Frequenz Energy-as-a-Service GmbH
+
+"""Methods for checking and clamping bounds and power values to exclusion bounds."""
```

Comment:
```suggestion
"""Utilities dealing with bounds, like checking and clamping, and power values to exclusion bounds."""
```

Reply:
It is actually clamping of (bounds and power values) to exclusion bounds.  So just changed to: 

> Utilities for checking and clamping bounds and power values to exclusion bounds.